### DISCUSSION TITLE
trying to use for the first time to pull ubuntu image

### DISCUSSION THREAD

**Discussion by mambon2** [bazel-contrib/rules_oci#465] [General]
I followed the examples but I'm getting this error.  anyone seen this before with bazel 5.0.0?

`@platforms//cpu:arm64 is not a valid select() condition for @ubuntu//:ubuntu.`

Here is the relevant code

```
  oci_image(
      name = "image",
      base = "@ubuntu",
      cmd = ["bee-stinger"],
      tars = ["stinger_pkg.tar"],
  )

  oci_pull(
      name = "ubuntu",
      image = "ubuntu",
      tag = "22.04",
      platforms = [
          "linux/arm64/v8",
          "linux/amd64",
      ],
)
```

**Comment by thesayyn** [comment]
We don't support anything before Bazel 6. You can however just use platform specific repo to work around this like `@ubuntu_linux_arm64_v8`.
